lib/liveValidation/operationValidator.ts

```diff
@@ -99,6 +99,7 @@ export const validateSwaggerLiveRequest = async (
   }
 
   const pathParam = extractPathParamValue(info.operationMatch!);
+  transformMapValue(pathParam, operation._pathTransform);
```

Comment:
what does this do?

Reply:
transform path parameters value per types. e.g. 'false' - > false.